contracts/test_contracts/blockinfo_tester/blockinfo_tester.cpp

```diff
@@ -1,4 +1,4 @@
-#include <blockinfo_tester/blockinfo_tester.hpp>
+#include "blockinfo_tester.hpp"
```

Comment:
I personally prefer using `<>` includes for public headers.  Is `blockinfo_tester.hpp` public?

Reply:
So do I. Not sure how that slipped in.